.github/workflows/checks.yml

```diff
@@ -28,3 +28,26 @@ jobs:
       - run: yarn
       - run: yarn run prettify:diff
       - run: yarn run lint
+  mobile-tests:
+    name: Mobile Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+          check-latest: true
+      - run: sudo apt-get update -y && sudo apt-get install rsync -y
```

Comment:
Remove this line -- you probably don't need it.  I bet whatever ubuntu image github is using will comes with the latest version of rsync.

Reply:
thanks, yeah I had seen this in https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md. I initially added it 1-to-1 from circle to see how much performance gains there were step to step, platform to platform. removing it now.